cranelift/filetests/filetests/isa/zkasm/add.clif

```diff
@@ -0,0 +1,38 @@
+test compile precise-output
+target sparc-unknown-unknown
+
+function u0:1(i32 vmctx) fast {
+    sig0 = (i32, i32, i32 vmctx) fast
+    fn0 = u0:0 sig0
+
+    block0(v0: i32):
+        v1 = iconst.i32 0x3b9a_c9ff
+        v2 = iconst.i32 0x3b9a_ca00
+        v3 = iadd v1, v2
+        v4 = iconst.i32 0x7735_93ff
+        call fn0(v3, v4, v0)
+        jump block1
+
+    block1:
+        return
+}
+
+; VCode:
+;     SP + 1 => SP
+;   sd RR,-1(sp)
+;     SP + 2 => SP
+;   sd B,-1(sp)
+; block0:
+;   
+;   +999999999 + +1000000000 => A;
+;   auipc B,0; ld B,12(B); j 12; .8byte 0x773593ff
+;   call userextname0
+;   j label1
+; block1:
+;   
+;   ld B,-1(sp)
+;     SP - 2 => SP
+;   ld RR,-1(sp)
+;     SP - 1 => SP
+;     :JMP(RR)
```

Comment:
NB: this is indeed rubbish. That’s the nature of the vcode though, and we should work on fixing it too :)